Tasks/JenkinsQueueJobV2/jenkinsqueuejobtask.ts

```diff
@@ -116,7 +113,6 @@ async function doWork() {
         tl.setVariable('JENKINS_JOB_ID', rootJob.ExecutableNumber.toString());
     } catch (e) {
         tl.debug(e.message);
-        tl._writeError(e);
```

Comment:
writeError was removed from the vsts-task-lib in an intermediate release.

Reply:
This was always an internal function.  You can just use `console.log`.  But in this case, I believe `setResult` will print the message anyway